paf-mvp-client-express/src/client-node.ts

```diff
@@ -450,6 +463,37 @@ export class ClientNode extends Node {
     }
   };
 
+  verifySeed = (req: Request, res: Response, next: NextFunction) => {
+    const message = fromDataToObject<SeedSignatureContainer>(req.body);
+    try {
+      const isResponseValid = this.client.verifySeed(message);
+      if (!isResponseValid) {
+        // TODO [errors] finer error feedback
+        const error: NodeError = {
+          type: 'VERIFICATION_FAILED',
+          details: '',
+        };
+        this.logger.Error(jsonProxyEndpoints.verifySeed, error);
+        res.status(400);
+        res.json(error);
+        next(error);
+      } else {
+        res.json(); // For the moment, send empty response
+        next();
+      }
+    } catch (e) {
+      this.logger.Error(jsonProxyEndpoints.verifySeed, e);
+      // FIXME finer error return
+      const error: NodeError = {
```

Comment:
No need to explicitly create an unknownError object. Calling the error handler with an internal error `next(e)` will do the work.

Reply:
It was previous to your commit ;-)

Reply:
You nedd to first merge your PR so that I can do this!

Reply:
Not really. ErrorHandler is defined in Node.ts and it's already good on the main branch.

Reply:
Yes, but that would have been inconsistent with the rest of the code: I prefer to merge everything in one go.
Now this is done, I let you validate, thanks!